contracts/lib/Liquidation.sol

```diff
@@ -0,0 +1,272 @@
+// SPDX-License-Identifier: BSD-3-Clause
+pragma solidity ^0.8.10;
+
+import { ExponentialNoError } from "./ExponentialNoError.sol";
+import { ComptrollerStorage } from "../ComptrollerStorage.sol";
+import { ComptrollerInterface } from "../ComptrollerInterface.sol";
+import { Comptroller } from "../Comptroller.sol";
+import { VToken } from "../VToken.sol";
+
+library Liquidation {
```

Comment:
I think this could be a contract, to be deployed externally and then set in the Comptroller with a new privileged function. That would allow us to update the Liquidation logic on runtime. Just to review it

Reply:
[c825a8e](https://github.com/VenusProtocol/isolated-pools/pull/535/commits/c825a8ec7c573578568d482e90b81e74ebf1ca59)